From 33afeae27524effcad92608da732ccc48e9cf6c7 Mon Sep 17 00:00:00 2001
From: Masahiro Yamada <masahiroy@kernel.org>
Date: Sat, 22 Jul 2023 13:47:49 +0900
Subject: [PATCH 07/62] kbuild: rpm-pkg: remove unneeded '-f $srctree/Makefile'
 in spec file

This is unneeded because the Makefile in the output directory wraps
the top-level Makefile in the srctree.

Just run $MAKE irrespective of the build location.

Signed-off-by: Masahiro Yamada <masahiroy@kernel.org>
---
 scripts/package/mkspec | 1 -
 1 file changed, 1 deletion(-)

diff --git a/scripts/package/mkspec b/scripts/package/mkspec
index c9299f9c1..a83b17b4a 100755
--- a/scripts/package/mkspec
+++ b/scripts/package/mkspec
@@ -12,7 +12,6 @@
 # how we were called determines which rpms we build and how we build them
 if [ "$1" = prebuilt ]; then
 	S=DEL
-	MAKE="$MAKE -f $srctree/Makefile"
 else
 	S=
 
-- 
2.42.0

